server/src/__tests__/server.test.ts

```diff
@@ -316,6 +316,30 @@ describe('server', () => {
     expect(result).toEqual([])
   })
 
+  it('responds to onCompletion with empty list when word is {', async () => {
+    const { connection, server } = await initializeServer()
+    server.register(connection)
+
+    const onCompletion = connection.onCompletion.mock.calls[0][0]
+
+    const result = await onCompletion(
+      {
+        textDocument: {
+          uri: FIXTURE_URI.ISSUE101,
+        },
+        position: {
+          // inside comment
```

Comment:
I guess this comment is a copy paste. Usually I write what is on the line and character position.

Reply:
whoops. you're right, my b. Fixed it